src/uct/ib/dc/dc_mlx5.h

```diff
@@ -197,11 +197,26 @@ typedef struct uct_dc_mlx5_ep_fc_entry {
 KHASH_MAP_INIT_INT64(uct_dc_mlx5_fc_hash, uct_dc_mlx5_ep_fc_entry_t);
 
 
+/* DCI pool
+ * there used same DCI stack to allocate and release DCI's:
+ * 
+ * +--------------+ +-------------+
+ * | to release   | | to allocate |
+ * +--------------+ +-------------+
+ * ^              ^ ^             ^
+ * |              | |             |
+ * 0        release stack      ndci
+ *              top top
+ * 
+ * Overall count of DCI's to relase and allocated DCI's could not be more than
+ * ndci and these stacks are not intersected
+ */
 typedef struct {
-    uint8_t       stack_top;                               /* dci stack top */
+    int8_t        stack_top;                               /* dci stack top */
     uint8_t       stack[UCT_DC_MLX5_IFACE_MAX_USER_DCIS];  /* LIFO of indexes of available dcis */
     ucs_arbiter_t arbiter;                                 /* queue of requests
                                                               waiting for DCI */
+    int8_t        release_stack_top;                       /* releasing dci's stack */
```

Comment:
what does a value of `-1` mean here? pls add comment

Reply:
> what does a value of `-1` mean here? pls add comment

added comment